Tasks/JenkinsQueueJobV2/task.json

```diff
@@ -110,6 +110,24 @@
             "required": false,
             "helpMarkDown": "Specifies strictness of a success definition: whether to consider unstable as a failure or not. False for non-strict, and true for strict version.",
             "groupName": "advanced"
+        },
+        {
+            "name": "retryCount",
+            "type": "string",
+            "label": "Number of retries for failed connection",
+            "defaultValue": "3",
+            "required": false,
+            "helpMarkDown": "Specify number of retries on errors or failures. Default is 3 times",
```

Comment:
There is no need to duplicate default value in description - could you remove it?

Reply:
Yeah, that has been overlooked. Better remove it.